tests/python/tests/test_pyiceberg.py

```
@@ -324,6 +324,164 @@ def test_write_read(namespace: conftest.Namespace):
+
+    table = catalog.load_table((*namespace.name, table_name))
+    assert table.properties["encryption.key-id"] == "master-key"
+
+    commit_url = _commit_table_url(namespace.warehouse, namespace.name, table_name)
+    headers = _auth_headers(namespace.warehouse)
+
+    # Add an encryption key
+    import base64
+
+    key_metadata = base64.b64encode(b"some-encrypted-key-bytes").decode()
+    resp = requests.post(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)